mmdet/datasets/structures.py

```diff
@@ -0,0 +1,292 @@
+import mmcv
+import numpy as np
+import pycocotools.mask as maskUtils
+import torch
+
+
```

Comment:
It may be better to move data structures into mmdet.cores since these structures are commonly used not only in data pipeline.